tools/user_migration/migrate_node.py

```diff
@@ -226,7 +239,8 @@ def conf_mysql(dsn):
         password=dsn.password,
         host=dsn.hostname,
         port=dsn.port or 3306,
-        database=dsn.path[1:]
+        database=dsn.path[1:],
+        auth_plugin="mysql_native_password"
```

Comment:
Honestly, I have no idea. I'm going to guess "yes" for now since things went pretty dead in the water without it when I tried against local and stage. We should flag @erkolson to make sure he's aware that this is present in case it catches him up. (he can either remove or comment the line as part of the work-around.)

Reply:
I have no idea what this does :-)

Reply:
in short, it says that the authentication model you will use to connect to mysql will be the native password one. That's the one that is usually default and matches how you'd connect if you were to use the mysql client. There are [other models](https://dev.mysql.com/doc/refman/8.0/en/authentication-plugins.html), if you were using one of them.

For inexplicable reasons, my local version switched this to "None" which prevented me from logging in at all. I'm not sure if it was a goof in the library or some new setting, so I decided to be explicit about it.

The fact that you have no idea is a good thing.